File: ActivityReservation.Models/ReservationPlace.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace ActivityReservation.Models
{
    [Table("tabReservationPlace")]
    public class ReservationPlace
    {
        /// <summary>
        /// 活动室id
        /// </summary>
        [Key]
        [Column]
        public Guid PlaceId { get; set; }

        /// <summary>
        /// 活动室名称
        /// </summary>
        [Column]
        [Required]
        [StringLength(16)]
        public string PlaceName { get; set; }

        /// <summary>
        /// 排序
        /// </summary>
        [Column]
        public int PlaceIndex { get; set; }

        /// <summary>
        /// 更新时间
        /// </summary>
        [Column]
        public DateTime UpdateTime { get; set; } = DateTime.UtcNow;

        /// <summary>
        /// 更新人
        /// </summary>
        [Column]
        [Required]
        public string UpdateBy { get; set; }

        /// <summary>
        /// 最多可预约时间段数量
        /// </summary>
        [Column]
        public int MaxReservationPeriodNum { get; set; }

        /// <summary>
        /// 是否启用
        /// </summary>
        [Column]
        public bool IsActive { get; set; } = true;

        /// <summary>
        /// 是否删除
        /// </summary>
        [Column]
        public bool IsDel { get; set; }

        [NotMapped]
        public IReadOnlyList<ReservationPeriod> ReservationPeriodList { get; set; }
    }
}

File: ActivityReservation.Clients/ReservationClient/src/app/models/ReservationPlace.ts
export class ReservationPlace{
    
    private _PlaceId : string;
    public get PlaceId() : string {
        return this._PlaceId;
    }
    public set PlaceId(v : string) {
        this._PlaceId = v;
    }
    
    
    private _PlaceName : string;
    public get PlaceName() : string {
        return this._PlaceName;
    }
    public set PlaceName(v : string) {
        this._PlaceName = v;
    }
    
    
    private _PlaceIndex : number;
    public get PlaceIndex() : number {
        return this._PlaceIndex;
    }
    public set PlaceIndex(v : number) {
        this._PlaceIndex = v;
    }
    
    
    private _MaxReservationPeriodNum : number;
    public get MaxReservationPeriodNum() : number {
        return this._MaxReservationPeriodNum;
    }
    public set MaxReservationPeriodNum(v : number) {
        this._MaxReservationPeriodNum = v;
    }
    
}